src/main/scala/uk/gov/nationalarchives/dp/client/DataProcessor.scala

```diff
@@ -370,6 +370,44 @@ class DataProcessor[F[_]]()(using me: MonadError[F, Throwable]) {
       version.stripPrefix("v").toFloat
     }
 
+  /** Returns a list of [[scala.xml.Node]] XML objects
+    *
+    * @param elem
+    *   The element containing the Entity's links
+    * @return
+    *   A `Seq` of `Node` containing the links
+    */
+  def getEntityLinksXml(elem: Elem): F[Seq[Node]] = me.pure((elem \ "Links" \ "Link"))
+
+  /** Returns a Seq containing multiple [[scala.xml.Node]] XML object objects representing an `EventAction`
+    *
+    * @param eventActionsResponseElement
+    *   The element containing the Event actions
+    * @return
+    *   A `Seq` of `EventAction` XML objects
```

Comment:
Could you update this to match the other descriptions, like "A `Seq` of `Node` containing event actions" or something like that.